client/galaxy/scripts/mvc/workflow/workflow-node.js

```diff
@@ -185,6 +186,12 @@ define(['mvc/workflow/workflow-view-node'], function( NodeView ) {
             var nodeView = node.nodeView;
             this.tool_state = data.tool_state;
             this.config_form = data.config_form;
+            if (this.config_form) {
+                this.tool_version = this.config_form.version;
```

Comment:
Can be a one-liner: `this.tool_version = this.config_form && this.config_form.version;`.